cmake/thirdparty/FindHIP/run_hipcc.cmake

```diff
@@ -0,0 +1,191 @@
+###############################################################################
```

Comment:
I dont know which license this is but please follow the BSD-3-Clause license.  cmake/thirdparty/FindROCm.cmake is a good example of a new file added by a contributor.

Reply:
Done